### DISCUSSION TITLE
How do I create folder names containing spaces and Sentence case text?

### DISCUSSION THREAD

**Discussion by ClaireHut42** [requarks/wiki#5237] [Help / Questions] [answered]
Hi, I have a couple of questions regarding folders that I hope you might be able to help me with?

1. How do I get a "folder" to display as "Release Notes" as seen on https://docs.requarks.io/en/releases/beta ? I have tried the kebab /release-notes/test-page and snake /release_notes/test-page naming convention hoping that this would be displayed differently in the navigation sidebar.
2. How have they been able to display the folder name "Release Notes" when its URL is actually /release/ on https://docs.requarks.io/en/releases/beta ?

Many thanks,
Claire

**Comment by NGPixel** [comment, marked as answer]
You need to create a page at the same path as the folder. For example, if you have these pages:

- /foo/bar/page1
- /foo/bar/page2
- /foo/bar/page3
- etc...

then create a new page at path `/foo` and another at `/foo/bar`

The title will be taken from both of these pages for the breadcrumbs.

**Reply by ClaireHut42** [reply]
Hi,

Wow! That worked! So basically the page has to have the same URL as the folder.

/foo is the folder
/foo is the page

The page title is 'Foo Bar' so 'Foo Bar' is displayed as the folder name.

Awesome. Thank you so much!
